lpa-submissions-e2e-tests/cypress/support/common/verifyQuestionTitle.js

```diff
@@ -1,11 +1,8 @@
 /// <reference types = "Cypress"/>
-import CommonPageObjects from '../PageObjects/CommonPageObjects';
-
-const commonPageObjects = new CommonPageObjects();
+import questionTitle from '../PageObjects/common-page-objects';
 
 module.exports = (question) => {
-  commonPageObjects
-    .questionTitle()
+  questionTitle()
```

Comment:
Is there value in this still being an import? Looks like it is only referred to in this method so an unnecessary abstraction?

Reply:
Pushed back.